kernel/src/main/scala/cats/kernel/Group.scala

```diff
@@ -11,13 +11,29 @@ trait Group[@sp(Int, Long, Float, Double) A] extends Any with Monoid[A] {
    * Find the inverse of `a`.
    *
    * `combine(a, inverse(a))` = `combine(inverse(a), a)` = `empty`.
+   *
+   * Example:
+   * {{{
+   * scala> import cats.implicits._
```

Comment:
The problem is that `cats.implicits` is only defined in `cats-core`, not in `cats-kernel`, so importing it here won't work.